Header header
bool markers_detected
uint16 markers_number  #all together, if more than two seen, placed from closer two to further
int8[] marker_id
float64[] marker_rotvec_x
float64[] marker_rotvec_y
float64[] marker_rotvec_z
float64[] marker_transvec_x
float64[] marker_transvec_y
float64[] marker_transvec_z
